common/settings/settings.ts

```diff
@@ -113,7 +113,7 @@ export interface KeyBindSet {
     readonly seekForward: KeyBind;
     readonly seekToPreviousSubtitle: KeyBind;
     readonly seekToNextSubtitle: KeyBind;
-    readonly seekToBeginningOfCurrentSubtitle: KeyBind;
+    readonly seekToBeginningOfCurrentOrPreviousSubtitle: KeyBind;
```

Comment:
Although it will be inconsistent with the behavior of the bind, could you keep the previous name of the KeyBind? Changing the key without some kind of migration will cause users to lose their setting.